libfuse/mount_test.go

```diff
@@ -2715,6 +2715,10 @@ func TestUnstageFile(t *testing.T) {
 	}
 }
 
+type TP struct {
```

Comment:
Was a residue of a debugging thing to get test printouts even in the event of a crash, removed.